### PR TITLE
feat: SupplementSecondaryFour

### PR DESCRIPTION
**Pull request by Geoff-Ford**
Slice layout, slice and tile and associated tests.

iPad Air (3rd Gen) Portrait
Screenshot 2020-09-01 at 16 40 13

iPad Air (3rd Gen) Landscape
Screenshot 2020-09-01 at 16 40 47

iPad Pro (11-inch) Portrait
Screenshot 2020-09-01 at 15 27 39

iPad Pro (11-inch) Landscape
Screenshot 2020-09-01 at 15 30 02

iPad Pro (12.9-inch) Portrait
Screenshot 2020-09-01 at 16 44 47

iPad Pro (12.9-inch) Landscape
Screenshot 2020-09-01 at 16 45 11

Android Pixel C Portrait
Screenshot 2020-09-01 at 17 38 33

Android Pixel C Landscape
Screenshot 2020-09-01 at 17 45 44

### CODE REVIEW

File: packages/edition-slices/src/slices/supplementsecondaryfour/index.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { SupplementSecondaryFourSlice } from "@times-components-native/slice-layout";
+import { TileAY } from "../../tiles";
+import { ResponsiveSlice } from "../shared";
+
+const SupplementSecondaryFour = (props) => {
```
Comment (c-j-j) [inline packages/edition-slices/src/slices/supplementsecondaryfour/index.js:7]: 🤔

File: packages/edition-slices/src/slices/supplementsecondaryfour/index.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  const render = (breakpoint) => {
+    const {
+      onPress,
+      slice: { isConsecutive, secondary1, secondary2, secondary3, secondary4 },
+    } = props;
+
+    return (
+      <SupplementSecondaryFourSlice
+        breakpoint={breakpoint}
+        isConsecutive={isConsecutive}
```
Comment (c-j-j) [inline packages/edition-slices/src/slices/supplementsecondaryfour/index.js:17]: Do you know why we're reversing the secondary 4 tile order here? I can see the original one does this - but I don't understand why we should reverse the article order that editorial have defined.

File: packages/edition-slices/__tests__/tile-ay/shared-tile-ay.base.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import "../mocks-tiles";
+import { editionBreakpoints } from "@times-components-native/styleguide";
+import { testTile } from "../shared-tile-utils";
+import { TileAY } from "../../src/tiles";
+
+export default () => {
+  describe("tile ar", () => {
```
Comment (c-j-j) [inline packages/edition-slices/__tests__/tile-ay/shared-tile-ay.base.js:7]: ar -> ay

**Review by c-j-j — APPROVED** [review]

File: packages/edition-slices/src/slices/supplementsecondaryfour/index.js (hunk 2 above)
Comment (Geoff-Ford) [reply packages/edition-slices/src/slices/supplementsecondaryfour/index.js:17]: 🤷  Don't know but this has to do everything the original does but just look different.

File: packages/edition-slices/src/slices/supplementsecondaryfour/index.js (hunk 1 above)
Comment (Geoff-Ford) [reply packages/edition-slices/src/slices/supplementsecondaryfour/index.js:7]: What? It isn't a class? 😄

Comment (c-j-j) [reply packages/edition-slices/src/slices/supplementsecondaryfour/index.js:7]: I copied this approach - definitely nicer

**Pull request merged**